frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-items.tsx

```diff
@@ -14,6 +14,55 @@ import { OperatorHubItemDetails } from './operator-hub-item-details';
 import { communityOperatorWarningModal } from './operator-hub-community-provider-modal';
 import { OperatorHubItem, InstalledState, ProviderType, CapabilityLevel } from './index';
 
+const canFilterByArchitecture = window.SERVER_FLAGS.GOARCH && window.SERVER_FLAGS.GOOS;
+
+const archBaseLabel = 'operatorframework.io/os.'; // for initial search as well
```

Comment:
I'm confuse. archBaseLabel ends in 'os.' and osBaseLabel ends in 'arch.'

Reply:
And thats how bugs are made. 😄 😄